main/pipeline/pipeline.cc

```diff
@@ -603,6 +607,10 @@ vector<ast::ParsedFile> index(core::GlobalState &gs, absl::Span<core::FileRef> f
     vector<ast::ParsedFile> ret;
     vector<ast::ParsedFile> empty;
 
+    cout << "***"
+         << pm_version()
+         << "***";
```

Comment:
This is just a way of testing that Prism is built correctly. I will remove it once we actually have prism parsing things.

<img width="773" alt="Screenshot 2024-05-21 at 4 21 37 PM" src="https://github.com/Shopify/sorbet/assets/9601737/6cd8e30f-e2dc-409c-b9d3-64141dd5f787">